src/cim/components/featureSupportLevels/ACMFeatureSupportLevelProvider.tsx

```diff
@@ -93,7 +93,7 @@ export const ACMFeatureSupportLevelProvider: React.FC<ACMFeatureSupportLevelProv
   const getDisabledReasonCallback = React.useCallback(
     (versionName: string, featureId: FeatureId) => {
       const isSupported = isFeatureSupportedCallback(versionName, featureId);
-      return getFeatureDisabledReason(featureId, undefined, isSupported);
+      return getFeatureDisabledReason(featureId, undefined, isSupported, (s: string) => s);
```

Comment:
you should pass `t` from `i18n`

```suggestion
      return getFeatureDisabledReason(featureId, undefined, isSupported, t);
```

```ts

import { useTranslation } from '../../../common/hooks/use-translation-wrapper';
...
const { t } = useTranslation();
...
return getFeatureDisabledReason(featureId, undefined, isSupported, t);
```

Reply:
done thank you